cobalt/browser/user_agent_platform_info.h

```diff
@@ -135,7 +140,8 @@ class UserAgentPlatformInfo : public web::UserAgentPlatformInfo {
   std::string evergreen_type_;
   std::string evergreen_file_type_;
   std::string evergreen_version_;
-  std::string android_os_experience_;  // Only via Client Hints
+  std::string android_build_fingerprint_;  // Only via Client Hints
```

Comment:
Could we have a more generic name for this? Maybe something like "firmware_version_ext" or "firmware_version_details" ? (And have a similar treatment for the actual header key)